osu.Game/Rulesets/Mods/ModCinema.cs

```diff
@@ -3,15 +3,43 @@
 
 using osu.Framework.Graphics.Sprites;
 using osu.Game.Graphics;
+using osu.Game.Rulesets.Objects;
+using osu.Game.Rulesets.UI;
+using osu.Game.Screens.Play;
 
 namespace osu.Game.Rulesets.Mods
 {
-    public class ModCinema : ModAutoplay
+    public abstract class ModCinema<T> : ModCinema, IApplicableToDrawableRuleset<T>
+        where T : HitObject
+    {
+        public virtual void ApplyToDrawableRuleset(DrawableRuleset<T> drawableRuleset)
+        {
+            drawableRuleset.SetReplayScore(CreateReplayScore(drawableRuleset.Beatmap));
+
+            drawableRuleset.Playfield.AlwaysPresent = true;
```

Comment:
Can you explain the reason behind this being required? I'm going through and trying to remove this requirement at the `Player` level.

Reply:
Required for hitsounds.